clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/EnableDisableAtomicOperationDescription.groovy

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.openstack.deploy.description.servergroup
+
+import com.netflix.spinnaker.clouddriver.deploy.DeployDescription
+import com.netflix.spinnaker.clouddriver.deploy.description.EnableDisableDescriptionTrait
+import com.netflix.spinnaker.clouddriver.openstack.deploy.description.OpenstackAtomicOperationDescription
+import groovy.transform.AutoClone
+import groovy.transform.Canonical
+
+@AutoClone
+@Canonical
+class EnableDisableAtomicOperationDescription extends OpenstackAtomicOperationDescription implements EnableDisableDescriptionTrait, DeployDescription {
```

Comment:
Because if you chose to use `OpenstackServerGroupAtomicOperationDescription` elsewhere, it would include `desiredPercentage` as a field which doesn't make sense as a part of a servergroup description.